subprojects/base-services/build.gradle.kts

```diff
@@ -54,4 +54,6 @@ packageCycles {
     excludePatterns.add("org/gradle/util/GradleVersion**")
 }
 
-jmh.includes.set(listOf("HashingAlgorithmsBenchmark"))
+jmh {
+    includes = listOf("HashingAlgorithmsBenchmark")
+}
```

Comment:
❔ Any reason to change this to multi-line?

Reply:
This is a workaround for: https://youtrack.jetbrains.com/issue/KT-56221/

Reply:
Reverted